.goreleaser.yml

```diff
@@ -17,6 +17,7 @@ builds:
       - windows
     goarch:
       - amd64
+      - arm
```

Comment:
Why not adding 386 as well?

Reply:
We haven't had a request for it. I am wary of adding more things to `goarch` without a request from users.

Reply:
That's fair :+1: